### PR TITLE
Pull Request for #50: Naming issue with Python Autofill

### PR DESCRIPTION
**Pull request by jinfu93**
Main addition is `source/lib/JMSLab/autofill.py` which includes @mcaceresb's `Autofill()` and a wrapper function to write the `.tex` file. 

Added a brief example of a use case under `issue50/`. 

@santiagohermo @veli-m-andirin @mcaceresb  if you have a to take a quick pass that'd be great, no worries if not! Do let me know if you have any thoughts on additional features that can improve flexibility.

@jmshapir upon completion here I'll update the module in IntQuantEc as well if that works.

Many thanks!

### CODE REVIEW

File: issue50/test_calcs.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import sys
```
[the remark this answers is not in the document]
Comment (santiagohermo) [reply issue50/test_calcs.py:1]: Looks like a great first pass @jmshapir @jinfu93, thanks @jinfu93! Happy to collaborate in the new issue :)

**Review by santiagohermo — APPROVED** [review]
I'm good here as well, thanks @jinfu93!

File: source/lib/JMSLab/autofill.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        
+        namespace = parent.f_locals
+    
+    commandname, content = var, namespace[var]
+    
+    return newcommand.format(commandname, content)
+
+def GenerateAutofillMacros(list, outfile, format):
+    autofill_file = open(outfile, 'w')
+    output_macros = ''.join(Autofill(var, format) for var in list)
```
[the remark this answers is not in the document]
Comment (santiagohermo) [reply source/lib/JMSLab/autofill.py]: Nice!

**Comment by jinfu93** [comment]
Summary for associated issue is [here](https://github.com/JMSLab/Template/issues/50#issuecomment-1079361117).

**Pull request merged**